test/values.ts

```diff
@@ -1209,7 +1209,7 @@ export const mockPositions: Position[] = [
         category: 'claimable',
         decimals: 18,
         network: 'celo',
-        balance: '0.098322815093446616',
+        balance: '0.098322815093446616', // total USD value = priceUsd * balance = $0.00009
```

Comment:
@jeanregisser is this a real scenario? i've got a claimable token on the `tokens` level, and another in the `tokens[0].tokens` level, so what would be displayed here is both these tokens as the reward?

Reply:
Good question, this wouldn't happen for real with Ubeswap farms, but I guess it's possible it could happen with more complex vault / yield farming.